packages/fxa-payments-server/server/lib/server.js

```diff
@@ -107,6 +107,7 @@ module.exports = () => {
     '/v1/products/:productId',
     '/v2/subscriptions',
     '/v2/products/:productId',
+    '/v1/checkout/:productId',
```

Comment:
I don't think we need the '/v1' part.

Reply:
Admittedly I was cargoculting -- but why is it there for the other routes?

Reply:
I guess I see without, with a v1 and a v2.

Reply:
Yeah, for a while we needed all those.  It should be safe to clean out the v1 and v2 stuff at this point.

Reply:
I'll do that here while I'm in here.